libs/fn/src/lib/select/option/option.component.ts

```diff
@@ -56,6 +56,13 @@ export class OptionComponent implements OnDestroy, FocusableOption {
     /** Whether this option is hidden */
     hidden = false;
 
+    /**
+     * The displayed value of the option. It shows the selected option in the select's trigger.
+     */
+    get viewValue(): string {
+        return (this.elementRef.nativeElement.textContent || '').trim();
```

Comment:
Not really a fan of this solution. Text content may change whenever developer wants, yet with this approach those changes won't be reflected. Take a look at core's select implementation. There's option to pass custom templateRef for rendering the selected value. Another option would be to use structural directive for the title, and return it's templateRef for rendering

```html
<fn-select placeholder="Select an option" label="Standard Selector">
    <li fn-option *ngFor="let option of options" [value]="option.value">
        <span *fnSelectTitle>{{ option.title }}</span>
    </li>
</fn-select>
```